src/test/java/uk/gov/ons/ctp/integration/rhsvc/cloud/GCSServiceIT_Test.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.ons.ctp.integration.rhsvc.cloud;
+
+import org.junit.Assert;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class GCSServiceIT_Test {
```

Comment:
**This currently prevents a mvn clean install due to the need to external service.**
Please merge in master - I have added skip integration tests to pom.
Then rename this class to end just '_IT' and it will be skipped unless we provide the right arg to mvn